source/deimos/openssl/evp.di

```diff
@@ -1239,7 +1239,7 @@ EVP_PKEY_CTX* EVP_PKEY_CTX_dup(EVP_PKEY_CTX* ctx);
 void EVP_PKEY_CTX_free(EVP_PKEY_CTX* ctx);
 
 int EVP_PKEY_CTX_ctrl(EVP_PKEY_CTX* ctx, int keytype, int optype,
-				int cmd, int p1, void* p2);
+				int cmd, int p1, const(void)* p2);
```

Comment:
It's not `const` in either the documentation or the C headers.

I don't think it can be `const`, because I see that some `get` macros calling it.